Add preview command line, format change, fix typo (#919)
github/view_component: docs/guide/previews.md

end
```

> Generate the preview using:

Reviewer: I think `generate` was the wrong word to use in the existing docs. It should instead say something like `which can then be accessed at:`

end
```

> You can also set a custom layout to be used by default for previews as well as the preview index pages via the `default_preview_layout` configuration option:

Reviewer: I _believe_ this line is correct as it stands, but could be written more clearly.

> You can also set a custom layout to be used by default for previews as well as preview the index pages via the `default_preview_layout` configuration option:

Reviewer: ```suggestion
To set a custom layout for previews and the previews index page, set: `default_preview_layout`:
```
```suggestion
You can also set a custom layout to be used by default for previews as well as preview the index pages via the `default_preview_layout` configuration option:
```
